nndt/space2/plot_tree.py

```diff
@@ -17,6 +18,14 @@
 )
 
 pyvista_pre_params = dict()
+updateparams = pyvista_pre_params.copy()
+result = set(updateparams).difference(set(pyvista_pre_params))
+if result:
+    pyvista_pre_params.update(result)
```

Comment:
Create a copy of just-created dictionary in this place doesn't solve the problem. Please add this code in "plot" function